docs/operations/README.md

```diff
@@ -45,7 +45,7 @@ Your sandbox space should've been setup as part of the onboarding process. If th
 
 ## Stable and Staging Release Rules
 
-Releases will be made for staging and stable every week starting on the first day of the sprint (Wednesday), with the second release of the sprint occuring halfway through the sprint. With the exception of first time going into production, these releases will NOT have the same code. The release to stable will be the same commit that was tagged for staging one week prior, making stable one week behind staging. Further, this means staging can be up to a week behind the main branch of code.
+Releases will be made for staging and stable twice a  week, ideally Tuesday and Thursday, but can be adjusted if needed. Code on `main` will be released to `staging`, and then on the following Tuesday/Thursday this `staging` release will become the new `stable` release. This means every release day, a release will be made to `stable` containing the last `staging` code. On this same day a new `staging` release will be made that contains the most up-to-date code on main. Thus, `staging` can be a few days behind the main branch, and `stable` will be a few days behind the code on `staging`.
```

Comment:
@kristinacyin, Cameron mentioned wanting a visual for this, do you have bandwidth for making a visual to represent this schedule?